core/src/main/java/org/arquillian/smart/testing/configuration/Scm.java

```diff
@@ -0,0 +1,43 @@
+package org.arquillian.smart.testing.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static org.arquillian.smart.testing.scm.ScmRunnerProperties.DEFAULT_LAST_COMMITS;
+import static org.arquillian.smart.testing.scm.ScmRunnerProperties.HEAD;
+import static org.arquillian.smart.testing.scm.ScmRunnerProperties.SCM_LAST_CHANGES;
+
+public class Scm implements ConfigurationSection {
+
+    private Range range;
+
+    public void setLastChanges(String lastChanges) {
```

Comment:
not used

Reply:
It's used by `ObjectMapper`

Reply:
now I got your point... interesting idea - I didn't think about it in this way. 👍

Reply:
add the annotation `@SuppressWarnings("unused")`